### PR TITLE
Documentation: Start production users documentation page

### PR DESCRIPTION
**Pull request by brancz**
@ant31 @fabxc @mxinden

### CODE REVIEW

File: Documentation/production-users.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Production users
+
+This document tracks people and use cases for etcd in production. By creating a list of production use cases we hope to build a community of advisors that we can reach out to with experience using various etcd applications, operation environments, and cluster sizes. The etcd development team may reach out periodically to check-in on how etcd is working in the field and update this list.
```
Comment (mxinden) [inline Documentation/production-users.md]: s/etcd/Prometheus operator/g

Comment (brancz) [reply Documentation/production-users.md]: oops :D good catch

**Comment by jolson490** [comment]
By the way, should a link to https://groups.google.com/forum/#!forum/prometheus-users and/or https://groups.google.com/forum/#!forum/prometheus-developers be added to https://github.com/coreos/prometheus-operator/blob/master/CONTRIBUTING.md#email-and-chat ?

Just curious, are there any other online spaces where users of prometheus-operator collaborate? (And there's always searching through Issues/PRs in this repo.)

**Comment by mxinden** [comment]
@jolson490 I feel like the *prometheus-users` and *prometheus-developers* Google group is a more general Prometheus upstream medium. #sig-instrumentation in the Kubernetes slack is probably a good place to mention.

**Comment by paulfantom** [comment]
Closing as obsolete after introducing ADOPTERS.md

**Pull request closed without merging**